Makefile.am

```diff
@@ -207,14 +216,30 @@ gresources += data/moltres/moltres.gresource
 pkgdata_DATA += $(gresources)
 
 # # # MAIN LIBRARY # # #
+BUILT_SOURCES = \
+	lib/eosknowledgeprivate/ekn-resources.h \
+	lib/eosknowledgeprivate/ekn-resources.c \
+	$(NULL)
+
+ekn_resources_deps = $(shell $(GLIB_COMPILE_RESOURCES) --generate-dependencies --sourcedir=lib/eosknowledgeprivate/ lib/eosknowledgeprivate/ekn-resources.gresource.xml)
```

Comment:
Use `:=` otherwise the command will be re-run every time the variable is referenced.